Doc/library/importlib.rst

```diff
@@ -1584,9 +1584,10 @@ Note that if ``name`` is a submodule (contains a dot),
 Importing a source file directly
 ''''''''''''''''''''''''''''''''
 
-``SourceFileLoader.load_module()`` has been deprecated -- this recipe should be used instead.
+This recipe should be used with caution: it is an approximation of an import statement where the file path is specified directly, rather than `sys.path` being searched.
```

Comment:
```suggestion
This recipe should be used with caution: it is an approximation of an import statement where the file path is specified directly, rather than :data:`sys.path` being searched.
```